observability/tools/Prometheus.md

```diff
@@ -0,0 +1,33 @@
+# Prometheus
+
+## Overview
+
+Originally built at SoundCloud, [Prometheus](https://prometheus.io/docs/introduction/overview/) is an open-source monitoring and alerting toolkit based on time series metrics data. It has become a de facto standard metrics solution in the cloud Native world and widely used with Kubernetes. 
+
+The core of Prometheus is a server that scrapes and stores metrics. There are other numerous optional features and components like an Alert-manager and [client libraries](https://prometheus.io/docs/instrumenting/clientlibs/) for programming languages to extend the functionalities of Prometheus beyond the basics. 
+The client libraries offer four [metric types](https://prometheus.io/docs/concepts/metric_types/); `Counter`, `Gauge`, `Histogram`, and `Summary`.
+
+## Why Prometheus?
+
+- Prometheus is a time series database and allow for events or measurements to be tracked, monitored, and aggregated over time. 
+- Prometheus is a pull based tool. One of the biggest advantages of Prometheus vs other monitoring tools is that Prometheus actively scrapes targets in order to retrieve metrics from them.
+- Prometheus allows for control over who to scrape, and how often to scrape them. Through the Prometheus server, there can be multiple scrape configurations, allowing for multiple rates for different targets.
+- Similar to Grafana, visualization fo the time series can be directly done through the Prometheus Web UI. The Web UI provides the ability to easily filter and have an overview of what is taking place with your different targets.
```

Comment:
Can we add some context (or even a link) for Grafana?

Reply:
Done